### ISSUE TITLE
npm install issue

### ISSUE THREAD

**Issue by dfcreative** [component/dialog#29]
Is it only me who face trouble installing dialog-component from npm?

```
npm install dialog-component
npm ERR! Failed to clone git+https://github.com/component/indexof#a3ef9ccef2b705f9688db5115c6698ab6ea49b98 from https://
github.com/component/indexof fatal: Could not switch to 'C:\Users\dmitry\AppData\Local\Temp\npm-6744-e23196d9\git-cache-
aab69fddfd6d\a3ef9ccef2b705f9688db5115c6698ab6ea49b98': Permission denied
npm ERR! Failed to clone git+https://github.com/component/indexof#a3ef9ccef2b705f9688db5115c6698ab6ea49b98 from https://
github.com/component/indexof
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "c:\\Program Files (x86)\\nodejs\\node.exe" "c:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-
cli.js" "install" "dialog-component"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code 128

npm ERR! Command failed: git -c core.longpaths=true clone C:\Users\dmitry\AppData\Roaming\npm-cache\_git-remotes\https-g
ithub-com-component-indexof-5ba90360 C:\Users\dmitry\AppData\Local\Temp\npm-6744-e23196d9\git-cache-aab69fddfd6d\a3ef9cc
ef2b705f9688db5115c6698ab6ea49b98
npm ERR! fatal: Could not switch to 'C:\Users\dmitry\AppData\Local\Temp\npm-6744-e23196d9\git-cache-aab69fddfd6d\a3ef9cc
ef2b705f9688db5115c6698ab6ea49b98': Permission denied
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     c:\Users\dmitry\Dropbox\Projects\test\npm-debug.log
```

**Comment by dfcreative** [comment]
It happend with `npm install classes-component`

**Comment by dfcreative** [comment]
Closed by aa5eb8a2
